3. Installation/7. Updating/README.md

```diff
@@ -1,6 +1,9 @@
 # Upgrading guides
 
-Ocassionally, no matter how hard we try.. We introduce changes that need a little extra action on your part. For normal upgrades please see the upgrade section of your chosen installation guide.
+Usually you can upgrade Rocket.Chat without much effort,
+for those changes check the _Upgrade section_ of your chosen installation guide
 
+Sometimes, no matter how hard we try, we introduce changes that
+might need some extra effort on your side: 
```

Comment:
Maybe here we should add suggestion to always read the release notes of a release before upgrading to it. 

We try to call major things out there